packages/compartment-mapper/test/scaffold.js

```diff
@@ -17,6 +17,25 @@ import { makeReadPowers } from '../src/node-powers.js';
 
 export const readPowers = makeReadPowers({ fs, crypto, url });
 
+const CompartmentInstrumentation = (() => {
```

Comment:
```suggestion
const compartmentInstrumentation = (() => {
```

Reply:
Ava runs tests concurrently. This should be a maker so tests do not have shared state.